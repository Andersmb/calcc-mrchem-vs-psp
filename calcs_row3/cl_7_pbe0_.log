Starting job 5688397 on c2-[30-31,47-48],c10-5 at Fri May 20 10:49:41 CEST 2022

Copying cl_7_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/doublets
Copying cl_7_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/doublets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688397      cl_7_pbe0_        180                                             00:53:45      0:0 
5688397.bat+      batch         36        1   00:00:03          0   00:00:03   00:53:45      0:0 
5688397.ext+     extern        180        5   00:00:00          4   00:00:00   00:53:45      0:0 
5688397.0      mrchem.x        180       30   00:53:28         29   02:26:37   00:53:33      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688397                                                                          
5688397.bat+     19280K          0     19280K        0              0          0 
5688397.ext+          0          4          0        0              4          0 
5688397.0     13106036K         29 5630217420        0             21          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688397                                                                                               
5688397.bat+       16.90M               0         16.90M        0.03M                0          0.03M 
5688397.ext+        0.00M               4          0.00M            0                4              0 
5688397.0           7.32M              26          6.85M      125.81M                1         71.03M 

Job 5688397 completed at Fri May 20 11:43:27 CEST 2022
