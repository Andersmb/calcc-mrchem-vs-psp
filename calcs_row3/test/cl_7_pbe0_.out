                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Cl            :        0.000000        0.000000        0.684104
    1       O            :        0.000000        2.381055       -0.757946
    2       O            :        0.000000       -2.381055       -0.757946
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              17
 Beta electrons          :                                              16
 Total electrons         :                                              33
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999923136e-01
    1    1    a          :                              9.999985005132e-01
    2    1    a          :                              9.999985035605e-01
    3    1    a          :                              9.999999870751e-01
    4    1    a          :                              9.999999999414e-01
    5    1    a          :                              9.999999997163e-01
    6    1    a          :                              1.000000000023e+00
    7    1    a          :                              1.000000385622e+00
    8    1    a          :                              1.000001064955e+00
    9    1    a          :                              9.999993509593e-01
   10    1    a          :                              1.000002269313e+00
   11    1    a          :                              9.999972436287e-01
   12    1    a          :                              1.000000210657e+00
   13    1    a          :                              1.000006497264e+00
   14    1    a          :                              1.000000809222e+00
   15    1    a          :                              9.999995187616e-01
   16    1    a          :                              1.000000310720e+00
   17    1    b          :                              9.999999682061e-01
   18    1    b          :                              9.999983675497e-01
   19    1    b          :                              9.999985078547e-01
   20    1    b          :                              9.999999870745e-01
   21    1    b          :                              9.999999999414e-01
   22    1    b          :                              9.999999997163e-01
   23    1    b          :                              1.000000000013e+00
   24    1    b          :                              1.000000385618e+00
   25    1    b          :                              1.000001065546e+00
   26    1    b          :                              9.999993485390e-01
   27    1    b          :                              1.000002276134e+00
   28    1    b          :                              9.999972491116e-01
   29    1    b          :                              1.000000209070e+00
   30    1    b          :                              1.000003985380e+00
   31    1    b          :                              1.000000816146e+00
   32    1    b          :                              9.999995089233e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                605.535534196967
 E-N energy              :            (au)              -1661.150701871969
 Coulomb energy          :            (au)                382.976439585439
 Exchange energy         :            (au)                -11.020853784132
 X-C energy              :            (au)                -34.200067710405
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                111.151454098935
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -717.859649584100
 Nuclear energy          :            (au)                111.151454098935
---------------------------------------------------------------------------
 Total energy            :            (au)             -6.067081954852e+02
                         :      (kcal/mol)             -3.807151375868e+05
                         :        (kJ/mol)             -1.592912136697e+06
                         :            (eV)             -1.650937032034e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -101.636951933908
    1    1    a          :            (au)                -18.975484199936
    2    1    a          :            (au)                -18.668335790229
    3    1    a          :            (au)                -10.018907047982
    4    1    a          :            (au)                 -7.767465276076
    5    1    a          :            (au)                 -7.761803594630
    6    1    a          :            (au)                 -7.760331570458
    7    1    a          :            (au)                 -1.334972234170
    8    1    a          :            (au)                 -1.063286509753
    9    1    a          :            (au)                 -0.804461086302
   10    1    a          :            (au)                 -0.701087935696
   11    1    a          :            (au)                 -0.506036778901
   12    1    a          :            (au)                 -0.398432921660
   13    1    a          :            (au)                 -0.298658120090
   14    1    a          :            (au)                 -0.238367638932
   15    1    a          :            (au)                 -0.148754502600
   16    1    a          :            (au)                 -0.112904772386
   17    1    b          :            (au)               -101.633178941474
   18    1    b          :            (au)                -18.966582301277
   19    1    b          :            (au)                -18.663218057627
   20    1    b          :            (au)                -10.015292097899
   21    1    b          :            (au)                 -7.765475641529
   22    1    b          :            (au)                 -7.758557346312
   23    1    b          :            (au)                 -7.748656708722
   24    1    b          :            (au)                 -1.315980561790
   25    1    b          :            (au)                 -1.047903644179
   26    1    b          :            (au)                 -0.783287528568
   27    1    b          :            (au)                 -0.564071554982
   28    1    b          :            (au)                 -0.489059729714
   29    1    b          :            (au)                 -0.385818446898
   30    1    b          :            (au)                 -0.228834227010
   31    1    b          :            (au)                 -0.154606772679
   32    1    b          :            (au)                 -0.142528373290
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -355.859293847658
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          5.744563e+00        -606.708195485165         -6.067082e+02
    1          5.047500e-01        -610.029356275142         -3.321161e+00
    2          2.400256e-01        -610.195499197755         -1.661429e-01
    3          1.318207e-01        -610.233122308969         -3.762311e-02
    4          6.798797e-02        -610.247301047382         -1.417874e-02
    5          8.930592e-02        -610.263471526471         -1.617048e-02
    6          2.534186e-02        -610.265649074259         -2.177548e-03
    7          2.705752e-02        -610.267335441911         -1.686368e-03
    8          1.476021e-02        -610.267671532377         -3.360905e-04
    9          1.692870e-02        -610.267921657662         -2.501253e-04
   10          9.675813e-03        -610.268073196194         -1.515385e-04
   11          1.145648e-02        -610.268184572970         -1.113768e-04
   12          7.079170e-03        -610.268232142284         -4.756931e-05
   13          6.180474e-03        -610.268269965653         -3.782337e-05
   14          6.822920e-03        -610.268308399031         -3.843338e-05
   15          7.707618e-03        -610.268337192905         -2.879387e-05
   16          5.228268e-03        -610.268350453329         -1.326042e-05
   17          3.651489e-03        -610.268361266775         -1.081345e-05
   18          2.802007e-03        -610.268367173486         -5.906711e-06
   19          1.889295e-03        -610.268370424757         -3.251270e-06
   20          1.625013e-03        -610.268372398768         -1.974011e-06
   21          1.191095e-03        -610.268373451099         -1.052332e-06
   22          4.564742e-04        -610.268373849002         -3.979021e-07
   23          3.187720e-04        -610.268374069123         -2.201216e-07
   24          2.978096e-04        -610.268374216209         -1.470855e-07
   25          2.865516e-04        -610.268374296955         -8.074585e-08
   26          1.503465e-04        -610.268374326624         -2.966908e-08
   27          9.431606e-05        -610.268374345408         -1.878448e-08
---------------------------------------------------------------------------
                      SCF converged in 27 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Cl            :        0.000000        0.000000        0.684104
    1       O            :        0.000000        2.381055       -0.757946
    2       O            :        0.000000       -2.381055       -0.757946
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                609.156736894421
 E-N energy              :            (au)              -1670.448216004267
 Coulomb energy          :            (au)                384.937968499956
 Exchange energy         :            (au)                -10.967384322090
 X-C energy              :            (au)                -34.098933512364
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                111.151454098935
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -721.419828444343
 Nuclear energy          :            (au)                111.151454098935
---------------------------------------------------------------------------
 Total energy            :            (au)             -6.102683743454e+02
                         :      (kcal/mol)             -3.829491835330e+05
                         :        (kJ/mol)             -1.602259384942e+06
                         :            (eV)             -1.660624771815e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -101.928385271974
    1    1    a          :            (au)                -19.285782526766
    2    1    a          :            (au)                -19.285664008476
    3    1    a          :            (au)                 -9.731776312920
    4    1    a          :            (au)                 -7.465226620284
    5    1    a          :            (au)                 -7.463291335988
    6    1    a          :            (au)                 -7.460832935870
    7    1    a          :            (au)                 -1.244707421432
    8    1    a          :            (au)                 -1.110036859410
    9    1    a          :            (au)                 -0.782267790689
   10    1    a          :            (au)                 -0.582081281774
   11    1    a          :            (au)                 -0.563984933351
   12    1    a          :            (au)                 -0.560764741095
   13    1    a          :            (au)                 -0.430916367358
   14    1    a          :            (au)                 -0.386978524284
   15    1    a          :            (au)                 -0.386273438786
   16    1    a          :            (au)                 -0.296056223865
   17    1    b          :            (au)               -101.925071990627
   18    1    b          :            (au)                -19.275535780182
   19    1    b          :            (au)                -19.275453413117
   20    1    b          :            (au)                 -9.728756751101
   21    1    b          :            (au)                 -7.463676754067
   22    1    b          :            (au)                 -7.459404754025
   23    1    b          :            (au)                 -7.451891897324
   24    1    b          :            (au)                 -1.229443542031
   25    1    b          :            (au)                 -1.090194194519
   26    1    b          :            (au)                 -0.751300241115
   27    1    b          :            (au)                 -0.555463486615
   28    1    b          :            (au)                 -0.548068876680
   29    1    b          :            (au)                 -0.536072802839
   30    1    b          :            (au)                 -0.378163809099
   31    1    b          :            (au)                 -0.374370900019
   32    1    b          :            (au)                 -0.360353845984
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -357.368249633666
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000001       -0.000516        1.205251
 Magnitude               :            (au)                        1.205251
                         :         (Debye)                        3.063442
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000       -0.497354
 Magnitude               :            (au)                        0.497354
                         :         (Debye)                        1.264148
---------------------------------------------------------------------------
 Total vector            :        0.000001       -0.000516        0.707897
 Magnitude               :            (au)                        0.707897
                         :         (Debye)                        1.799295
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 53m 30s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
